### DISCUSSION TITLE
下载歌曲时卡在0%

### DISCUSSION THREAD

**Discussion by xiaye6783** [Lost-MSth/Arcaea-server#126] [Q&A]
请问下下载歌曲时突然卡在0%不动是什么原因
也检查了songs目录和songlist没有问题

**Comment by Lost-MSth** [comment]
网络问题排除了吗，如果是卡在 0% 过一会发生未知错误，那也有可能是下载地址的自动获取有问题，可以在配置文件里手动设置地址

https://github.com/Lost-MSth/Arcaea-server/wiki/Q&A#why-not-success-in-downloading-songs-or-charts-为什么歌曲或谱面下载失败
